components/Payment/interfaces/fee_calculator_interfaces.rst

```diff
@@ -0,0 +1,26 @@
+Fee Calculator Interfaces
+=========================
+
+Fee Calculator Interface
```

Comment:
Used it more as a title than actual name, agree that it looks a tad weird on the page, but gorgeous in the menu.